path_oidc.go

```diff
@@ -216,9 +217,15 @@ func (b *jwtAuthBackend) pathCallback(ctx context.Context, req *logical.Request,
 		if oidcReq.idToken == "" {
 			return logical.ErrorResponse(errLoginFailed + " No code or id_token received."), nil
 		}
+
+		// Verify the ID token received from the authentication response.
 		rawToken = oidc.IDToken(oidcReq.idToken)
+		if _, err := provider.VerifyIDToken(ctx, rawToken, oidcReq); err != nil {
```

Comment:
Nice catch, were we not doing this previously?

Reply:
We were prior to adopting cap. Glad I took a closer look at this code path..